frontend/src/layouts/NavigationBar/NavigationDrawer.tsx

```diff
@@ -50,24 +41,44 @@ const NavigationDrawer: React.FC<Props> = (props) => {
               component={NavLink}
               to={item.link}
               sx={{
-                textAlign: "center",
-                color: "#fff",
+                textAlign: "left",
+                color: "black",
                 "&.active": {
-                  bgcolor: "#B83B39" // Darker shade for active
+                  color: "#B83B39" // Darker shade for active
                 },
                 "&:hover": {
-                  bgcolor: "#C84B4A" // Slightly darker for hover effect
-                }
+                  bgcolor: "#d5d3d2" // Slightly darker for hover effect
+                },
+                minHeight: 60
               }}
             >
               <ListItemText
                 primary={item.text}
-                primaryTypographyProps={{ fontSize: "1rem", color: "#fff" }}
+                primaryTypographyProps={{
+                  fontSize: "1.5rem", // Increased font size
```

Comment:
Increased from something still relevant(?) Also, when making UI changes, test pictures would be bonus